packages/api/src/index.ts

```diff
@@ -162,18 +166,28 @@ const getApp = async () => {
 				res.status(500).send(msg);
 				return;
 			}
-			const parsedItem = TranscriptionItem.safeParse(item);
+			const parsedItem = TranscriptionDynamoItem.safeParse(item);
 			if (!parsedItem.success) {
 				const msg = `Failed to parse item ${exportRequest.data.id} from dynamodb. Error: ${parsedItem.error.message}`;
 				console.error(msg);
 				res.status(500).send(msg);
 				return;
 			}
+			const transcriptText = await getObjectText(
```

Comment:
I think it's worth considering forgoing dynamo altogether and just using s3 to save ourselves a request here and in the output handler. the s3 object has the user's email in its metadata and can be found with just a message id.

Reply:
I guess a counterargument is that storing transcriptKeys in dynamo gives us a bit more flexibility to change the file structure in s3

Reply:
This is a good point - the main reason we added dynamo was to store the transcript text. However, I think it's useful to have all the various bits of metadata associated with a transcript in one place as e.g. we can use it for audting or for building a nicer UI. We could dump the entire json blob, metadata and all in S3 as well though.

I think it's worth having a DB, if we wanted to implement a progress bar for example, it would be much nicer doing it with dynamo as it allows you to update a single property rather than having to overwrite the whole file. 

I've worked on a project before where they didn't bother with a database and just used S3 for everything. It definitely worked - the tools has been in use for years - but debugging was always a faff as you were constantly downloading random files rather than being able to query a database. Dynamo is particularly nice as we can just see all the data in the browser https://eu-west-1.console.aws.amazon.com/dynamodbv2/home?region=eu-west-1#item-explorer?maximize=true&operation=SCAN&table=transcription-service-PROD